### DISCUSSION TITLE
Manage multiple wallboxes

### DISCUSSION THREAD

**Discussion by vincenzosuraci** [lbbrhzn/ocpp#746] [Q&A] [answered]
I was wondering if it is possible to control 2 (or more) wallboxes at the same time with the same integration? In case I have 2 wallboxes connected to the server, they will both appear as two separate entities in Home Assistant, so that I can control them separately?

Many thanks for any advise on this feature.

Regards,
Vincenzo

**Comment by lbbrhzn** [comment, marked as answer]
Yes, this is possible. Currently it requires multiple ocpp instances: you need to create a separate ocpp server for each charger to connect to.  In the future we might have a version of the ocpp that supports multiple chargers to a single server.

**Reply by vincenzosuraci** [reply]
Thanks for the swift reply.

I'm studying your (wonderful!) code and, if I'm not wrong, the current limitations are:
1. OCPP 2.0.1 is not supported
2. it is not possible to manage 3-phases charge points
3. it is not possible to manage a charge point having multiple ( > 1) connectors

**Reply by RafSchandevyl** [reply]
Any info on when this could be possible with one Server ?

**Reply by fredriklj** [reply]
> Any info on when this could be possible with one Server ?

Bump on this question, and in particular bullet number three on managing a charge point with more than one connector. 

Is this in the road map in the foreseeable future?

**Comment by lbbrhzn** [comment]
1) and 3) are indeed not possible in the current version.

However, 2) is possible. I have a three phase charge myself.

Kind regards,

Lbbrhzn

**Comment by RafSchandevyl** [comment]
How can i configure it at this moment.  I added a second ocpp server with port 0.0.0.0:9001  but i can't connect to it.
